packages/logger/README.md

```diff
@@ -0,0 +1,34 @@
+# X Logger
+
+This project is a simple logger
+
+### Installation
+
+You can install the logger using NPM:
+
+```
+npm install @empathyco/x-logger --save-dev
+```
+
+### Usage
+
+```
+import { logger, LogLevel } from '@empathyco/x-logger';
+```
+
+There are different log levels to be set in the console level and server level.
+
+```
+logger.consoleLevel = LogLevel.warn;
```

Comment:
Should we add the whole list abouth the different values that you can use??

export enum LogLevel {  silent = 0,  error = 1,  warn = 2,  info = 3,  debug = 4,  trace = 5}

Should we indicate in some way that when you are setting the consoleLevel to warn, you will print lvls 0, 1 and 2. I know that its logic but i can help to the users to know which levels are going to be printed each time that you set the level.

Reply:
Go ahead if you can, i think that will be useful

Reply:
Done @tiborux